changelog/updates/2025-04-28-weekly-updates.md

```diff
@@ -0,0 +1,33 @@
+- SDK: rust ([1.85.1](https://github.com/rust-lang/rust/releases/tag/1.85.1) (includes [1.85.0](https://github.com/rust-lang/rust/releases/tag/1.85.0)))
+- azure, dev, gce, sysext-python: gdbm ([1.25](https://lists.gnu.org/archive/html/info-gnu/2025-03/msg00010.html))
+- base, dev: azure-vm-utils ([0.6.0](https://github.com/Azure/azure-vm-utils/releases/tag/v0.6.0) (includes [0.5.2](https://github.com/Azure/azure-vm-utils/releases/tag/v0.5.2), [0.5.1](https://github.com/Azure/azure-vm-utils/releases/tag/v0.5.1), [0.5.0](https://github.com/Azure/azure-vm-utils/releases/tag/v0.5.0)))
+- base, dev: bind ([9.18.31](https://bind9.readthedocs.io/en/v9.18.31/notes.html#notes-for-bind-9-18-31) (includes [9.18.30](https://bind9.readthedocs.io/en/v9.18.30/notes.html#notes-for-bind-9-18-30)))
+- base, dev: elfutils ([0.192](https://inbox.sourceware.org/elfutils-devel/CAJDtP-T3+gXqHWp3T0mejWWbPr0_1tHetEXwfB67-o+zz7ShiA@mail.gmail.com/T/))
+- base, dev: ethtool ([6.11](https://git.kernel.org/pub/scm/network/ethtool/ethtool.git/tree/NEWS?h=v6.11))
+- base, dev: iproute2 ([6.14.0](https://lore.kernel.org/all/20250324092319.28d39f2f@hermes.local/))
+- base, dev: libnvme ([1.12](https://github.com/linux-nvme/libnvme/releases/tag/v1.12))
+- base, dev: libxcrypt ([4.4.38](https://github.com/besser82/libxcrypt/releases/tag/v4.4.38) (includes [4.4.37](https://github.com/besser82/libxcrypt/releases/tag/v4.4.37)))
+- base, dev: nvme-cli ([2.12](https://github.com/linux-nvme/nvme-cli/releases/tag/v2.12))
+- base, dev: open-iscsi ([2.1.11](https://github.com/open-iscsi/open-iscsi/blob/2.1.11/Changelog))
+- base, dev: open-isns ([0.103](https://github.com/open-iscsi/open-isns/releases/tag/v0.103))
+- base, dev: pkgconf ([2.4.3](https://github.com/pkgconf/pkgconf/blob/pkgconf-2.4.3/NEWS) (includes [2.4.2](https://github.com/pkgconf/pkgconf/blob/pkgconf-2.4.2/NEWS), [2.4.1](https://github.com/pkgconf/pkgconf/blob/pkgconf-2.4.1/NEWS), [2.4.0](https://github.com/pkgconf/pkgconf/blob/pkgconf-2.4.0/NEWS)))
+- base, dev: rpcbind ([1.2.7](http://git.linux-nfs.org/?p=steved/rpcbind.git;a=shortlog;h=refs/tags/rpcbind-1_2_7))
+- base, dev: timezone-data ([2025b](https://lists.iana.org/hyperkitty/list/tz-announce@iana.org/thread/6JVHNHLB6I2WAYTQ75L6KEPEQHFXAJK3/))
+- base, dev: xfsprogs ([6.13.0](https://web.git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git/tree/doc/CHANGES?h=v6.13.0))
+- dev: eselect ([1.4.30](https://gitweb.gentoo.org/proj/eselect.git/tree/NEWS?h=eselect-1.4.30))
+- dev: mpfr ([4.2.2](https://www.mpfr.org/mpfr-4.2.2/))
+- docker: docker-buildx ([0.21.2](https://github.com/docker/buildx/releases/tag/v0.21.2) (includes [0.21.1](https://github.com/docker/buildx/releases/tag/v0.21.1), [0.21.0](https://github.com/docker/buildx/releases/tag/v0.21.0)))
```

Comment:
`s/docker:/sysext-docker:/`?

Reply:
Good point, I'll fix that in the automation too. I think it treats sysexts in rootfs differently from sysexts to be downloaded.

Reply:
Done.